x-pack/qa/sql/src/main/resources/agg.sql-spec

```diff
@@ -456,6 +456,13 @@ selectHireDateGroupByHireDate
 SELECT hire_date HD, COUNT(*) c FROM test_emp GROUP BY hire_date ORDER BY hire_date DESC;
 selectSalaryGroupBySalary
 SELECT salary, COUNT(*) c FROM test_emp GROUP BY salary ORDER BY salary DESC;
+selectSalaryGroupBySalaryHavingIsNotNull
+SELECT salary, COUNT(*) c FROM test_emp GROUP BY salary HAVING MAX(languages) IS NOT NULL ORDER BY salary DESC;
```

Comment:
This is a good test atm, but not really testing the behavior as MAX() doesn't _seem_ to return `null` in any scenario https://github.com/elastic/elasticsearch/issues/34896

Reply:
Good catch - due the use of primitives, a non-null value is returned all the time.